lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
Yeah, I think just set `/var/sufia` as the default location. If they don't override it, they'll get an error when it can't be created.  We could also add a method to the initializer that validates the location exists and raise an exception at that point.